src/transformers/models/yolos/modeling_yolos.py

```diff
@@ -84,10 +84,6 @@ class YolosObjectDetectionOutput(ModelOutput):
             values are normalized in [0, 1], relative to the size of each individual image in the batch (disregarding
             possible padding). You can use [`~YolosImageProcessor.post_process`] to retrieve the unnormalized bounding
             boxes.
-        auxiliary_outputs (`list[Dict]`, *optional*):
```

Comment:
Let's keep this in - even if it wasn't available in the original implementation it's been part of the model for a while, so we'd had to do a whole deprecation cycle to remove and AFAICT it's just an additional feature

Reply:
reverted!